docs/concepts/conda_pypi.md

```diff
@@ -0,0 +1,102 @@
+Pixi is built on top of both the conda and PyPI ecosystems.
+
+**Conda** is a cross-platform, cross-language package ecosystem that allows users to install packages and manage environments.
+It is widely used in the data science and machine learning community, but it is also used in other fields.
+It’s power comes from the fact that it always installs binary packages, meaning that it doesn’t need to compile anything.
+This makes the ecosystem very fast and easy to use.
+
+**PyPI** is the Python Package Index, which is the main package index for Python packages.
+It is a much larger ecosystem than conda, especially because the boundary to upload packages is lower.
+This means that there are a lot of packages available, but it also means that the quality of the packages is not always as high as in the conda ecosystem.
+
+Pixi **can** install packages from **both** **ecosystems**, but it uses a **conda-first approach**.
+
+The simplified process is as follows:
+
+1. Resolve the conda dependencies.
+2. Map the conda packages to PyPI packages.
+3. Resolve the [remaining]("All pypi dependencies already installed as conda dependencies will not be fetched from PyPI") PyPI dependencies.
```

Comment:
Not sure what this link is supposed to do :)

Reply:
This is a tooltip

Reply:
It behaves like a link though

Reply:
That's unfortunately how tooltips work, i thought was good like this. Otherwise I would just remove the information.